src/main/scala/intellij/haskell/action/DownloadCabalConfigFile.scala

```diff
@@ -0,0 +1,17 @@
+package intellij.haskell.action
+
+import com.intellij.openapi.actionSystem.{AnAction, AnActionEvent}
+import intellij.haskell.external.component.HaskellComponentsManager.refreshGlobalProjectInfo
+import intellij.haskell.external.component.PathComponent
+
```

Comment:
Cool.
>Yes, that's okay.

Yes, currently the restart repl action will also trigger the config file check [here](https://github.com/rikvdkleij/intellij-haskell/blob/master/src/main/scala/intellij/haskell/action/RestartStackReplsAction.scala#L67), but restart repl will take a relatively long time, and the completion of package names did not reply on other things, so maybe extract the cabal file downloading action is more flexible for users to use?
>Well, then it's better to create or reuse existing action which refreshes everything. We could use the action which restarts the repls for this.

I think the change of Stack LTS is perceptible here cause this action will remove the old cabal config file (if it exists) and then download a new one.
>We have many dependencies. Also user can change dependending libraries or Stack LTS.

Right, we have `DownloadLibrarySources` already, so maybe another download action here is also reasonable?
>We also have action for downloading sources.

Take you time 😄 , if you still not happy with this action, I'll remove it then.
>I will think about it...